src/data/alacritty.yml

```diff
@@ -0,0 +1,46 @@
+Type: desktop-application
+ID: io.alacritty.Alacritty
+Package: alacritty
+Name: 
+  C: Alacritty
+ProjectLicense: APACHE-2.0
+MetadataLicense: APACHE-3.0
+Summary: 
+  C: A cross-platform, GPU enhanced terminal emulator.
+Description:
+  C: >-
+    <p>Alacritty is the fastest terminal emulator in existence. Using the GPU for rendering enables optimizations that simply aren't possible without it.</p>
+    <p>Alacritty currently supports macOS, Linux, BSD, and Windows.</p>
+    <p>Alacritty is a terminal emulator with a strong focus on simplicity and performance</p>
+DeveloperName:
+  C: Joe Wiln
+Categories:
+- terminal
+Keywords:
+  C:
+  - terminal
+  - GPU 
+  - emulator
+Icon:
+  cached:
+  - name: alacritty.png
+    width: 64
+    height: 64
+  - name: alacritty.png
+    width: 128
+    height: 128
+Url:
+  homepage: https://github.com/jwilm/alacritty
+  bugtracker: https://github.com/jwilm/alacritty/blob/master/CONTRIBUTING.md#bug-reports
+Screenshots:
+  - thumbnails: []
+    source-image:
+      url: https://cloud.githubusercontent.com/assets/4285147/21585004/2ebd0288- 
```

Comment:
This screenshot URL here isn't valid, so it isn't appearing in the shop.

Reply:
https://cloud.githubusercontent.com/assets/4285147/21585004/2ebd0288-d06c-11e6-95d3-4a2889dbbd6f.png

That should help! I'm not sure how that happened...